.github/composite-actions/major-version-upgrade-util/action.yml

```diff
@@ -39,11 +43,13 @@ runs:
         migration_mode: ${{ inputs.migration_mode }}
         pg_old_dir: ${{ inputs.pg_old_dir }}
         pg_new_dir: ${{ inputs.pg_new_dir }}
+        server_collation_name: ${{ inputs.server_collation_name }}
 
     - name: Run Verify Tests
       if: always() && steps.run-pg_upgrade.outcome == 'success' && inputs.is_final_ver == 'true'
       uses: ./.github/composite-actions/run-verify-tests
       with:
         is_final_ver: ${{ inputs.is_final_ver }}
         pg_new_dir: ${{ inputs.pg_new_dir }}
-        migration_mode: ${{ inputs.migration_mode }}
\ No newline at end of file
+        migration_mode: ${{ inputs.migration_mode }}
+        server_collation_name: ${{ inputs.server_collation_name }}
```

Comment:
Nit: let's have a newline at the end of the file.

Reply:
done

Reply:
done